frontend/src/app/testResults/EmailTestResultModal.tsx

```diff
@@ -45,9 +47,13 @@ export const EmailTestResultModal = ({ closeModal, testResultId }: Props) => {
               {formatFullName(patient)}'s test result from{" "}
               {formatDateLong(dateTested)} will be sent to the following emails:
             </div>
-            {patient.emails?.map((email: string) => (
-              <div key={email}>{email}</div>
-            ))}
+            <ul className="usa-list usa-list--unstyled">
+              {patient.emails?.map((email: string) => (
+                <li className="line-height-sans-2" key={email}>
```

Comment:
⚠️ Chose to use a uswds provided class for the line height that was close to the original code as possible instead of adding a custom css class to make the line heights identical. (The difference is 0.8px -- new is 19.2px while the old is 18.4px -- which I feel is negligible for the benefit of not adding a custom css class)